### PR TITLE
Add 1-19 main post submit

### PR DESCRIPTION
**Pull request by bnrjee**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: jobs/aws/eks-distro/main-1-19-postsubmits.yaml [hunk 1]
```diff
@@ -40,9 +39,7 @@ postsubmits:
         - bash
         - -c
         - >
-          make release -C projects/kubernetes/kubernetes DEVELOPMENT=false RELEASE_BRANCH=1-19 IMAGE_TAG='$(GIT_TAG)-$(PULL_BASE_SHA)'
-          &&
-          mv ./projects/kubernetes/kubernetes/_output/1-19/* /logs/artifacts
+          make postsubmit-conformance DEVELOPMENT=false RELEASE_BRANCH=1-19 IMAGE_TAG='$(GIT_TAG)-$(PULL_BASE_SHA)' ARTIFACT_BUCKET=artifactsstack-3794122512-artifactsbucket2aac5544-1f3dgu9wrpiz2
```
Comment (vivek-koppuru) [inline jobs/aws/eks-distro/main-1-19-postsubmits.yaml:42]: Do we need to have this named that way? If our postsubmit is anyways going to run conformance, then I think we can keep the name as postsubmit

Comment (vivek-koppuru) [reply jobs/aws/eks-distro/main-1-19-postsubmits.yaml:42]: For the ARTIFACT_BUCKET, do we need to hard code it here? Also, I thought this postsubmit was not going to call the artifacts account?

Comment (bnrjee) [reply jobs/aws/eks-distro/main-1-19-postsubmits.yaml:42]: Will remove the bucket name in the next iteration and hard code the name in the make file.

**Pull request merged**